steps/set-ios-version/2.0.0/step.yml

```diff
@@ -0,0 +1,70 @@
+title: Set iOS Info.plist - Bundle Version and Bundle Short Version String
+summary: Sets the Bundle Version and Bundle Short Version String to the specified
+  value, in the target Info.plist file for the next build.
+description: Sets the Bundle Version and Bundle Short Version String to the specified
+  value, in the target Info.plist file for the next build.
+website: https://github.com/BOOMik/steps-set-ios-version
+source_code_url: https://github.com/BOOMik/steps-set-ios-version
+support_url: https://github.com/BOOMik/steps-set-ios-version/issues
+published_at: 2017-11-27T19:16:44.634166503+03:00
+source:
+  git: https://github.com/BOOMik/steps-set-ios-version.git
+  commit: 199824aae51d4f7b029ff41b17876b9da1b0d0ad
+host_os_tags:
+- osx-10.10
+project_type_tags:
+- ios
+type_tags:
```

Comment:
please normalize the type tags as mentioned in the step.yml : https://github.com/BOOMik/steps-set-ios-version/blob/master/step.yml#L29

in general: please pick one type only

this property is only used for categorization, the best is if you select only a single one, but in any case please select only from the list mentioned there, to make it easier to search & discover steps in the library :)

Reply:
Our recommendation would be `utility`, but you're of course free to choose another one if you disagree :)